src/build/build-libgerbil.ss

```diff
@@ -107,14 +107,14 @@
 
 (def (fold-options opt stdlib-spec base)
   (let lp ((rest stdlib-spec) (result base))
-    (match rest
-      ([spec . rest]
-       (cond
-        ((pget opt (cdr spec))
-         => (lambda (val)
-              (lp rest (string-append result " " val))))
-        (else (lp rest result))))
-      (else result))))
+      (match rest
+	([spec . rest]
+	 (cond
+	  ((pget opt (cdr spec))
+	   => (lambda (val)
+		(lp rest (string-append result " " val))))
+	  (else (lp rest result))))
+	(else result))))
```

Comment:
what happened here?

Reply:
I'll blame OSXIII and that I didn't have gerbil mode loaded. Fixing now.